### ISSUE TITLE
npm start encounters error

### ISSUE THREAD

**Issue by vikubin** [marceloaugusto80/electron-react-typescript-boilerplate#21]
**Describe the bug**
npm start encounters error

**To Reproduce**
1. Clone the repository
2. Run: 'npm install'
3. Run: 'npm start'
4. See error:
```
C:\Users\XXXXX\WEB_Sync\electron-react-typescript-boilerplate>npm start

> electron-react-typescript-boilerplate@2.1.0 start C:\Users\XXXXX\WEB_Sync\electron-react-typescript-boilerplate
> npx concurrently 'npm run start:server' 'npm run start:electron' --kill-others --kill-others-on-fail

[0] ''npm' is not recognized as an internal or external command,
[0] operable program or batch file.
[1] 'run' is not recognized as an internal or external command,
[1] operable program or batch file.
[3] ''npm' is not recognized as an internal or external command,
[3] operable program or batch file.
[4] 'run' is not recognized as an internal or external command,
[4] operable program or batch file.
[3] 'npm exited with code 1
--> Sending SIGTERM to other processes..
[1] run exited with code 1
--> Sending SIGTERM to other processes..
[0] 'npm exited with code 1
--> Sending SIGTERM to other processes..
[4] run exited with code 1
--> Sending SIGTERM to other processes..
[2] start:server' exited with code 1
--> Sending SIGTERM to other processes..
[5] start:electron' exited with code 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-react-typescript-boilerplate@2.1.0 start: `npx concurrently 'npm run start:server' 'npm run start:electron' --kill-others --kill-others-on-fail`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electron-react-typescript-boilerplate@2.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\XXXXX\AppData\Roaming\npm-cache\_logs\2021-07-05T15_42_05_915Z-debug.log
```

Log File: 
[2021-07-05T15_42_05_915Z-debug_redacted.txt](https://github.com/marceloaugusto80/electron-react-typescript-boilerplate/files/6765337/2021-07-05T15_42_05_915Z-debug_redacted.txt)

**Expected behavior**
Successful launch of dev server and electron

**Desktop (please complete the following information):**
 - OS: Windows 10

**Comment by vikubin** [comment]
Fix in PR: #22

**Comment by marceloaugusto80** [comment]
@philku solved the issue. 

We should escape the double quotes like \" and not use single quotes.

Anyways, @philku solution was much cleaner and clever.

start script is working now.


Thanks, @philku!
